OpusTestForSeeking/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Concentus.Oggfile;
+using Concentus.Structs;
+using System;
+using System.IO;
+
+namespace OpusTestForSeeking
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string opusFile = "opus.opus";
+            string outFile = "out.raw";
+            string out2File = "out2.raw";
+
+            using (var fileIn = new FileStream(opusFile, FileMode.Open))
```

Comment:
You could convert this into a unit test fairly easily by putting the input file into a  `[DeploymentItem("TestData/opus.opus")]`, piping the output to a memory stream, and then asserting that the first snippet of decoded audio actually matches what you would expect to hear after the seek